flink-end-to-end-tests/test-scripts/test_kubernetes_pyflink_application.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
```

Comment:
I think you need to start the `test_kubernetes_application.sh` in `run-nightly-tests.sh`.